docs/building-modules/query-services.md

```diff
@@ -21,7 +21,7 @@ Let us break it down:
 
 - The `path` is an array of `string`s that contains the type of the query, and that can also contain `query` arguments. See [`queries`](./messages-and-queries.md#queries) for more information.
 - The `req` itself is primarily used to retrieve arguments if they are too large to fit in the `path`. This is done using the `Data` field of `req`.
-- The [`Context`](../core/context.md) contains all the necessary information needed to process the `query`, as well as a cache-wrapped copy of the latest state. It is primarily used by the [`keeper`](./keeper.md) to access the state.
+- The [`Context`](../core/context.md) contains all the necessary information needed to process the `query`, as well as a branch of the latest state. It is primarily used by the [`keeper`](./keeper.md) to access the state.
```

Comment:
in my mental model, the state is a tree, and "branch of the state" to me refers to a subset of the tree.

Can we use "snapshot" here too? following https://github.com/cosmos/cosmos-sdk/pull/8102#discussion_r551289050

Reply:
I'm not using snapshot anywhere else. I proposed it in that comment but when going through docs _branch_ was more adequate for me. If we want to change it then lets continue in the other thread.